test/metabase/xrays/automagic_dashboards/interesting_test.clj

```diff
@@ -33,70 +32,6 @@
       (throw (ex-info (format "Did not find %s.%s" (name table) (name column))
                       {:table table :column column}))))
 
-(deftest ^:parallel field-matching-predicates-test
```

Comment:
Why to remove whole test and not just `ga:` related parts?

Reply:
good catch! Fixed